src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.tasktypes.Task;
+import duke.ui.Ui;
+
+
```

Comment:
Minor comment: Extra breakline is inserted

Reply:
Thanks for highlighting. Will fix!